### PR TITLE
bundle install and refactored loop in the jobs controller

### PR DESCRIPTION
**Pull request by superechnik**
@orosenberg +1 my changes!

### CODE REVIEW

File: app/controllers/jobs_controller.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@ def index
 @jobs = Job.all
   
 
-@results = Job.all.each do
+@results = @jobs.each do
 	   |job| job.schedule
 	   end
```
Comment (orosenberg) [inline app/controllers/jobs_controller.rb]: are you creating an array of job.schedule output here?
or are you assigning the loop to the results? @superechnik

**Comment by superechnik** [comment]
@orosenberg I was trying to create an array of job messages.  While Job.first.schedule outputs the message, looping through jobs with each outputs only the Job object.  This was updated using @jobs.collect --> block in a later update.

File: app/controllers/jobs_controller.rb (hunk 1 above)
Comment (orosenberg) [inline app/controllers/jobs_controller.rb]:
```
@results = []
@jobs.each do |job|
    temporaryVar = job.schedule
   @results.push(temporaryVar)
end

**Comment by orosenberg** [comment]
+1

**Pull request merged**